cmd/booster-http/util.go

```diff
@@ -9,3 +16,29 @@ func addCommas(count uint64) string {
 	}
 	return str
 }
+
+// checkPaymentChannelBalance checks a payment channel balance and returns true if the AmountPaid is greater than the expected amount
+func checkPaymentChannelBalance(rpcClient *rpc.RpcClient, paymentChannelId types.Destination, expectedAmount *big.Int) bool {
+	if rpcClient == nil {
+		panic("the rpcClient is nil")
```

Comment:
I guess this panic is justified by "it should never happen"? Perhaps we should warn about the panic in the godoc?